src/Sections/Scores.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import './sass.scss';
+
+export default class Scores extends React.Component {
```

Comment:
why class component?

Reply:
Though this component is static we can make it a pure component.
Else, convert it to functional one.

Reply:
changed to PureComponent